demo/src/TreeViewFinderConfig.jsx

```diff
@@ -29,7 +30,7 @@ import {
  * @param {EventListener}   onSelectionTypeChange - onChange type EventListener on the multiselect value change.
  * @param {EventListener}   onOnlyLeavesChange - onChange type EventListener on the onlyLeaves value change.
  */
-const TreeViewFinderConfig = (props) => {
+function TreeViewFinderConfig(props) {
```

Comment:
why this change?

Reply:
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/function-component-definition.md